darts/models/torch_forecasting_model.py

```diff
@@ -164,6 +166,9 @@ def __init__(self,
         torch_device_str
             Optionally, a string indicating the torch device to use. (default: "cuda:0" if a GPU
             is available, otherwise "cpu")
+        force
+            force model loading, even if the data does not exist. Note that you won't be able to train model initialized
```

Comment:
I think you should update this docstring. Something like this?
```
If set to `True`, any previously-existing model with the same name will be reset (all checkpoints will be discarded).
```

Reply:
done.